src/app/not-found.js

```diff
@@ -1,5 +1,30 @@
 const error = () => {
-  return <div>error</div>;
+  return (
+    <div className=" flex flex-col my-20 items-center">
+      <p className="text-hlg-orange font-bold font-serif text-center text-4xl">
+        {" "}
+        Whoops!{" "}
+      </p>
+
+      <p className="text-hlg-blue-300 text-center font-bold font-serif text-8xl ">
+        {" "}
+        404{" "}
+      </p>
+
+      <p className=" text-hlg-orange font-bold font-mono text-center text-3xl">
+        {" "}
+        Page Not Found{" "}
+      </p>
+
+      <a
```

Comment:
use Next.js <Link> instead of <a>

Reply:
What do you mean by use Next.js instead of ?